apps/new/widget/page/projects/Editor.jsx

```diff
@@ -503,27 +503,35 @@ function onCreateProject() {
 const following = Social.get(`${context.accountId}/graph/follow/*`);
 const followingAccountSuggestion = following && Object.keys(following);
 
+const StyledTypeahead = styled.div`
```

Comment:
Is this a style that will be used across all Typeahead components? Should it go in CSS?